distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -129,6 +129,10 @@ log4j2.appender.osgi-karafConsole.type = PaxOsgi
 log4j2.appender.osgi-karafConsole.name = osgi-karafConsole
 log4j2.appender.osgi-karafConsole.filter = VmLogAppender
 
+# SSHD logger
```

Comment:
❓ Should this be added to the `log4j.xml` configuration file as well?

Reply:
I don't really understand why this is needed in the first place ([KARAF-5559](https://issues.apache.org/jira/browse/KARAF-5559)) 🤷‍♀️. It might be worth adding a comment here about why these lines are needed.

Reply:
I just added it to suppress the logging as Karaf is now doing. spifly is suppressed for the same reason https://github.com/codice/ddf/blob/211864351d183ed7f423b4f502689b9143619fbf/distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg#L93-L94.  I feel like a comment might be overkill in this instance.

Reply:
I know it was added in Karaf 4.2.1, but I don't understand why. A comment would help prevent someone from removing these seemingly-useless lines in the future. E.g. "The The org.apache.aries.spifly level was set to WARN in Karaf 4.2.1. Do not remove."